app/views/layouts/application.html.slim

```diff
@@ -1,12 +1,11 @@
-- content_for :template, yield(:title) || t(:loans)
-
 - content_for :head
   = javascript_include_tag "application"
   = stylesheet_link_tag "application", :media => "all"
-  = csrf_meta_tags
+  // = csrf_meta_tags
+
+- content_for :template, yield(:title)
 
 - content_for :content
-  = render 'alerts'
```

Comment:
Seems like it would be good to have some kind of alerts system in the layout. Did this not work? It looks like it was moved to `app/views/admin/application`. Maybe it should be moved back out of the admin subfolder? We might have to play around a bit to figure out how to get it working.